dlhub_sdk/client.py

```diff
@@ -218,14 +218,30 @@ def run(self, name, inputs,
         funcx_id = self.fx_cache[name]
         payload = {'data': inputs}
         task_id = self._fx_client.run(payload, endpoint_id=self.fx_endpoint, function_id=funcx_id)
-        #r = self.post(servable_path, json_body=data)
-        # if (not asynchronous and r.http_status != 200) \
-        #         or (asynchronous and r.http_status != 202):
-        #     raise Exception(r)
 
         # Return the result
         return DLHubFuture(self, task_id, async_wait).result() if not asynchronous else task_id
 
+    def run_serial(self, servables, inputs, async_wait=5):
+        """Invoke each servable in a serial pipeline.
+        This function accepts a list of servables and will run each one,
+        passing the output of one as the input to the next.
+
+        Args:
+             servables (list): A list of servable strings
+             inputs: Data to pass to the first servable
+             asycn_wait (float): Seconds to wait between status checks
+        Returns:
+            Results of running the servable
+        """
+        if not isinstance(servables, list):
+            print("run_serial requires a list of servables to invoke.")
```

Comment:
Should this be a `ValueError` instead to halt execution of any program that passes bad inputs?

Reply:
ah missed that. I agree it should raise an exception. Maybe an `assert` would be enough also